src/test/regress/expected/undistribute_table_cascade.out

```diff
@@ -376,7 +387,7 @@ BEGIN;
   set citus.multi_shard_modify_mode to 'sequential';
   ALTER TABLE partitioned_table_1_100_200 ADD CONSTRAINT non_inherited_fkey FOREIGN KEY(col_1) REFERENCES partitioned_table_2_100_200(col_1);
   SELECT undistribute_table('partitioned_table_2', cascade_via_foreign_keys=>true);
-ERROR:  cannot cascade operation via foreign keys as partition table undistribute_table_cascade.partitioned_table_1_100_200 involved in a foreign key relationship that is not inherited from it's parent table
+ERROR:  cannot cascade operation via foreign keys as partition table undistribute_table_cascade.partitioned_table_2_100_200 involved in a foreign key relationship that is not inherited from it's parent table
```

Comment:
I guess it's pretty normal to see this output change as we changed where we check for non-inherited fkeys on partitions